src/fread.c

```diff
@@ -426,12 +426,12 @@ static const char* filesize_to_str(const size_t fsize)
     }
     if (ndigits == 0 || (fsize == (fsize >> shift << shift))) {
       if (i < sizeof(suffixes)) {
-        snprintf(output, sizeof(output), "%"PRIu64"%cB (%"PRIu64" bytes)", // # notranslate
-                 (fsize >> shift), suffixes[i], fsize);
+        snprintf(output, sizeof(output), "%"PRIu64"%ciB (%"PRIu64" bytes)", // # notranslate
```

Comment:
tempted to say we should "just" put the full prefix in `suffixes`, but it means moving from simple one-byte-each `char` array to 3-byte version, probably much more trouble than it's worth.